app/(gcforms)/[locale]/(form administration)/form-builder/[id]/responses/[statusFilter]/actions.ts

```diff
@@ -240,6 +261,35 @@ export const getSubmissionsByFormat = async ({
             } as Answer;
           }
 
+          // Handle "Split" AddressComplete in a similiar manner to dynamic fields.
+          if (
+            question?.type === FormElementTypes.addressComplete &&
+            question.properties.full === false
```

Comment:
Same comment as before

```
question.properties.full 
```

This should be tied back to address I can't tell looking at this code what full is

Reply:
Missed that one!~ It should of been .addressComponents?.splitAddress instead of .full.